Revise blog submission guidelines as per the blog subproject (#22176)
kubernetes/website: content/en/docs/contribute/new-content/blogs-case-studies.md

Anyone can write a blog post and submit it for review.
Case studies require extensive review before they're approved.

<!-- body -->

## The Kubernetes Blog

The Kubernetes blog is used by the project to communicate new features, community reports, and any news that might be relevant to the Kubernetes community. 
This includes end users AND developers. 
Generally speaking the blog tends to skew towards things that happen in the core project, however the community is always interested in things that are happening in the Kubernetes ecosystem, so we encourage you to consider submitting content to the blog. 

Anyone can write a blog post and submit it for review.

### Guidelines and expectations

> 

Reviewer: Maybe also remind contributors that blog content must be original?

https://github.com/kubernetes/website/blob/master/LICENSE allows you to write a blog article, have it published, and then repeat it elsewhere. But publishing in the opposite order is not allowed.

- Blog posts should not be vendor pitches. 
  - Articles must contain content that applies broadly to the Kubernetes community - For example, a submission should focus on upstream Kubernetes as opposed to vendor-specific configurations.

>   - Links should primarily be to the official Kubernetes documentation. When using external references, links should be diverse - For example a submission shouldn't contain only links back to a single company's blog.

Reviewer 2: @castrojo - it might be helpful to put a link here to the Documentation Style Guide and "what's allowed" 

https://kubernetes.io/docs/contribute/style/content-guide/#what-s-allowed

  - This can sometime be a delicate balance. The [blog team](https://kubernetes.slack.com/messages/sig-docs-blog/) is there to give guidance on whether a post is appropriate for the Kubernetes blog, so don't hesistate to reach out. 
- Blog posts do not offer deadlines.

>     - Articles are reviewed by community volunteers and _cannot guarantee_ any time-based deadlines, though we will try our best to accomodate you.

Reviewer 2: ```suggestion
    - Articles are reviewed by community volunteers and **can not guarantee** any time-based deadlines, though we will try our best to accomodate you.
```

Reviewer 2: Made this bold, because it's fun to live **BOLD** :joy:

  - Many core parts of the project are submitting posts during release windows and publication time will be delayed. Consider submitting during a quieter period of the release cycle.

>   - If you are looking for greater coordination on post release dates, coordinating with [CNCF marketing] is a more appropriate choice than submitting a blog post.

Reviewer 2: ```suggestion
  - If you are looking for greater coordination on post release dates, coordinating with [CNCF marketing](https://www.cncf.io/about/contact/) is a more appropriate choice than submitting a blog post.
```

Reviewer 2: Added a link to the CNCF "contact us" page

  - The components of Kubernetes are purposely modular, so tools that use existing integration points like CNI and CSI are on topic. 
  - Posts about other CNCF projects may or may not be on topic. We recommend asking the blog team before submitting a draft.
    - Many CNCF projects have their own blog. These are often a better choice for posts. There are times of major feature or milestone for a CNCF project that users would be interested in reading on the Kubernetes blog.
- Blog posts should aim to be future proof
  - Given the development velocity of the project, we want evergreen content, that won't require updates to stay accurate to the reader. 
  - It can be a better choice to add a tutorial or update official documentation, then write a higher level overview as a blog post.
    - Consider concentrating the long technical content as a call to action of the blog post, and focus on the problem space or why readers should care.

### Technical Considerations for submitting a blog post

> Submissions need to be in Markdown format to be used by the [Hugo](https://gohugo.io/) generator for the blog. There are [many resources available] on how to use this technology stack.

Reviewer 2: ```suggestion
Submissions need to be in Markdown format to be used by the [Hugo](https://gohugo.io/) generator for the blog. There are many resources available on how to use this technology stack.
```

Reviewer 2: Due to the third-party content suggestion, I hesitated to add in any links here, but we could link to things via the Hugo site or one of the "Awesome Project" repos, otherwise!

Reviewer: Hugo is essential third party software in as much as the contents of the website repo aren't much use without Hugo.

Reviewer 2: Good point - I think this qualifies as "...if it's necessary for Kubernetes to function."

Reviewer 2: Updated with a link to the Hugo docs around getting started ✅

Submissions need to be in Markdown format to be used by the [Hugo](https://gohugo.io/) generator for the blog. There are [many resources available](https://gohugo.io/documentation/) on how to use this technology stack.

We recognize that this requirement makes the process more difficult for less-familiar folks to submit, and we're constantly looking at solutions to lower this bar. If you have ideas on how to lower the barrier, please volunteers to help out. 

The SIG Docs [blog subproject](https://github.com/kubernetes/community/tree/master/sig-docs/blog-subproject) manages the review process for blog posts. For more information, see [Submit a post](https://github.com/kubernetes/community/tree/master/sig-docs/blog-subproject#submit-a-post).

To submit a blog post follow these directions:

- [Open a pull request](/docs/contribute/new-content/new-content/#fork-the-repo) with a new blog post. New blog posts go under the [`content/en/blog/_posts`](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts) directory.

- Ensure that your blog post follows the correct naming conventions and the following frontmatter (metadata) information:

>   - The Markdown file name must follow the format `YYYY-MM-DD-Your-Title-Here.md`. For example, `2020-02-07-Deploying-External-OpenStack-Cloud-Provider-With-Kubeadm.md`.

Reviewer 3: do *NOT* include dots in the file name.

Author: I added a warning bullet, thanks!

>   - The Markdown file name must follow the format `YYYY-MM-DD-Your-Title-Here.md`. For example, `2020-02-07-Deploying-External-OpenStack-Cloud-Provider-With-Kubeadm.md`.

Reviewer 2: ```suggestion
  - The Markdown file name must follow the format `YYYY-MM-DD-Your-Title-Here.md`. For example, `2020-02-07-Deploying-External-OpenStack-Cloud-Provider-With-Kubeadm.md`.
  - Do **not** include dots in the filename. A name like `2020-01-01-whats-new-in-1.19.md` will causes failures during a build.
```

  - Articles must contain content that applies broadly to the Kubernetes community - For example, a submission should focus on upstream Kubernetes as opposed to vendor-specific configurations. Check the [Documentation style guide](https://kubernetes.io/docs/contribute/style/content-guide/#what-s-allowed) for what is typically allowed on Kubernetes properties. 
  - Links should primarily be to the official Kubernetes documentation. When using external references, links should be diverse - For example a submission shouldn't contain only links back to a single company's blog.
  - This can sometime be a delicate balance. The [blog team](https://kubernetes.slack.com/messages/sig-docs-blog/) is there to give guidance on whether a post is appropriate for the Kubernetes blog, so don't hesistate to reach out. 
- Blog posts do not offer deadlines.
    - Articles are reviewed by community volunteers and **can not guarantee** any time-based deadlines, though we will try our best to accomodate you.
  - Many core parts of the project are submitting posts during release windows and publication time will be delayed. Consider submitting during a quieter period of the release cycle.

>   - If you are looking for greater coordination on post release dates, coordinating with [CNCF marketing](https://www.cncf.io/about/contact/) is a more appropriate choice than submitting a blog post.

Author: Yeah, sometimes people want things orchestrated on a specific day to coordinate with their own marketing efforts, the idea here is to let them know that the CNCF has resources for that.

>   - If you are looking for greater coordination on post release dates, coordinating with [CNCF marketing](https://www.cncf.io/about/contact/) is a more appropriate choice than submitting a blog post.

Reviewer 4: @castrojo then maybe:

```suggestion
  - If you want to coordinate a blog post release with external marketing efforts from your organization, contact  [CNCF marketing](https://www.cncf.io/about/contact/).
```

  - Topics related to participation in or results of Kubernetes SIGs activities are always on topic (see the work in the [Upstream Marketing Team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/blog-guidelines.md#upstream-marketing-blog-guidelines) for support on these posts). 
  - The components of Kubernetes are purposely modular, so tools that use existing integration points like CNI and CSI are on topic. 
  - Posts about other CNCF projects may or may not be on topic. We recommend asking the blog team before submitting a draft.
    - Many CNCF projects have their own blog. These are often a better choice for posts. There are times of major feature or milestone for a CNCF project that users would be interested in reading on the Kubernetes blog.
- Blog posts should be original content
    - The official blog is not for repurposing existing content from a third party as new content.
    - The [license](https://github.com/kubernetes/website/blob/master/LICENSE) for the blog does allow commercial use of the content for commercial purposes, just not the other way around. 
- Blog posts should aim to be future proof

>   - Given the development velocity of the project, we want evergreen content, that won't require updates to stay accurate to the reader. 

Reviewer 4: ```suggestion
  - Given the development velocity of the project, we want evergreen content that won't require updates to stay accurate for the reader. 
```

>   - It can be a better choice to add a tutorial or update official documentation, then write a higher level overview as a blog post.

Reviewer 4: ```suggestion
  - It can be a better choice to add a tutorial or update official documentation than to write a high level overview as a blog post.
```

    - Consider concentrating the long technical content as a call to action of the blog post, and focus on the problem space or why readers should care.

### Technical Considerations for submitting a blog post

Submissions need to be in Markdown format to be used by the [Hugo](https://gohugo.io/) generator for the blog. There are [many resources available](https://gohugo.io/documentation/) on how to use this technology stack.

> We recognize that this requirement makes the process more difficult for less-familiar folks to submit, and we're constantly looking at solutions to lower this bar. If you have ideas on how to lower the barrier, please volunteers to help out. 

Reviewer 4: ```suggestion
We recognize that this requirement makes the process more difficult for less-familiar folks to submit, and we're constantly looking at solutions to lower this bar. If you have ideas on how to lower the barrier, please volunteer to help out. 
```

The SIG Docs [blog subproject](https://github.com/kubernetes/community/tree/master/sig-docs/blog-subproject) manages the review process for blog posts. For more information, see [Submit a post](https://github.com/kubernetes/community/tree/master/sig-docs/blog-subproject#submit-a-post).

To submit a blog post follow these directions:

- [Open a pull request](/docs/contribute/new-content/new-content/#fork-the-repo) with a new blog post. New blog posts go under the [`content/en/blog/_posts`](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts) directory.

- Ensure that your blog post follows the correct naming conventions and the following frontmatter (metadata) information:

  - The Markdown file name must follow the format `YYYY-MM-DD-Your-Title-Here.md`. For example, `2020-02-07-Deploying-External-OpenStack-Cloud-Provider-With-Kubeadm.md`.

>   - Do **not** include dots in the filename. A name like `2020-01-01-whats-new-in-1.19.md` will causes failures during a build.

Reviewer 4: ```suggestion
  - Do **not** include dots in the filename. A name like `2020-01-01-whats-new-in-1.19.md` causes failures during a build.
```

>   - Do NOT include dots in the filename, this will break the renderer. Pay close attention when dealing with versions in your filename and slug, for example `1.15-is-great.md` will break, `1-15-is-great.md` is correct.

Reviewer 2: ```suggestion
```

>   - The front matter must include the following:

Reviewer 5: This is a repeat :x

> ## {{% heading "whatsnext" %}}

Reviewer: Is there anything after this line?

Author: No, this was on the initial page so I just left it here. Should I remove it?

Reviewer: Not a big deal but nice to remove it.
